control/control.rnc

```diff
@@ -534,9 +534,16 @@ upgrade = element upgrade {
             )
           }*
       }? &
-      element compatible_vendors {
-          LIST,
-          element compatible_vendor { STRING }*
+      element product_upgrade {
```

Comment:
Ah, I think we should allow list of hashes here to allow defining more product upgrades. E.g. allow different vendors for `Leap` -> `Jump` upgrade and for (hypothetical) `SLES` -> `Jump` upgrade.